classes/Product.php

```diff
@@ -501,6 +499,8 @@ class ProductCore extends ObjectModel
             ],
 
             /* Shop fields */
+            'id_supplier' => ['type' => self::TYPE_INT, 'shop' => true, 'validate' => 'isUnsignedId'],
```

Comment:
```suggestion
            'id_supplier' => ['type' => self::TYPE_INT, 'shop' => true, 'validate' => 'isUnsignedId'],
```

Looks like a BC Break. Is it for v9 or 8.1?

Reply:
The problem is that the default supplier's price and product's wholesale price are synced now, and since wholesale price is a multishop value it's the only way we can handle a logical synchronisation in multishop

So I don't know if it's a BC break but I know I don't have other solution for this 🤷‍♂️